apps/dc_tools/odc/apps/dc_tools/utils.py

```diff
@@ -195,7 +182,7 @@ def index_update_dataset(
     auto_add_lineage: Optional[bool] = False,
     publish_action: Optional[str] = None,
     stac_doc: Optional[dict] = None,
-) -> int:
+):
```

Comment:
@omad thanks for fixing this. In this case the type annotation was wrong, but for people who aren't familiar with the code base it's good to have return type annotations to inform what the function is expected to return, so it's nice if you replace faulty return types with the right ones in the future.